### PR TITLE
style(metric): small stylistic adjustments

### PR DESCRIPTION
**Pull request by markov00**
## Summary
This PR adjusts titles weights on the metric visualization and reduces a bit one size breakpoint.
cc @gvnmagni 

### Checklist

- [x] The proper **chart type** label has been added (e.g. `:xy`, `:partition`)
- [x] The proper **feature** labels have been added (e.g. `:interactions`, `:axis`)
- [x] Unit tests have been added or updated to match the most common scenarios
- [x] Visual changes have been tested with all available themes including `dark`, `light`, `eui-dark` & `eui-light`

### CODE REVIEW

**Comment by markov00** [comment]
buildkite update screenshots

**Review by gvnmagni — APPROVED** [review]

**Comment by markov00** [comment]
buildkite update screenshots

File: packages/charts/src/chart_types/metric/renderer/dom/text.tsx [hunk 1]
```diff
@@ -121,15 +122,15 @@ function elementVisibility(
...
         visibilityBreakpoint.titleMaxLines,
         textMeasure,
       ),
       subtitleLines: wrapText(
         datum.subtitle ?? '',
-        subtitleFont,
+        SUBTITLE_FONT,
         SUBTITLE_FONT_SIZE[size],
         maxTitlesWidth,
         visibilityBreakpoint.subtitleMaxLines,
```
Comment (nickofthyme) [inline packages/charts/src/chart_types/metric/renderer/dom/text.tsx:136]: Is this right? The titles are all reading the fontSize 🤔 

image

Comment (markov00) [reply packages/charts/src/chart_types/metric/renderer/dom/text.tsx:136]: ups

**Pull request merged**